packages/compass-e2e-tests/index.js

```diff
@@ -11,16 +11,20 @@ const {
   buildCompass,
 } = require('./helpers/compass');
 const { createUnlockedKeychain } = require('./helpers/keychain');
+const ResultLogger = require('./helpers/result-logger');
 
 const keychain = createUnlockedKeychain();
 
+const LOG_PATH = path.resolve(__dirname, '.log');
```

Comment:
Can we export it from compass helper or something instead so we don't hardcode the same path all over the place?